lib/nc_time_axis/__init__.py

```diff
@@ -211,3 +198,8 @@ def convert(cls, value, unit, axis):
             raise ValueError('A "calendar" attribute must be attached to '
                              'netcdftime object to understand them properly.')
         return netcdftime.date2num(value, cls.standard_unit, calendar=first_value.calendar)
+
+
+# Register NetCDFTimeConverter with matplotlib.unit's converter dictionary. 
+if netcdftime.datetime not in munits.registry:
+    munits.registry[netcdftime.datetime] = NetCDFTimeConverter()
```

Comment:
I'm not sure what to put as the comment here. Everything I come up just sounds like a rewording of what the code below it says.
I tried to work off the wording from [here](http://matplotlib.org/api/units_api.html#module-matplotlib.units)